source/includes/gcp/_instance_groups.md

```diff
@@ -1 +1,225 @@
-### Instance groups
\ No newline at end of file
+### Instance groups
+An instance group is a collection of VM instances that you can manage as a single entity
+
+<!-------------------- LIST INSTANCE GROUPS -------------------->
+
+#### List instance groups
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/instancegroups"
+
+# The above command returns JSON structured like this:
+```
+```json
+{
+  "data": [
+    {
+      "creationTimestamp": "2019-09-06T12:28:47.389-07:00",
+      "description": "",
+      "fingerprint": "42WmSpB8rSM=",
+      "kind": "compute#instanceGroup",
+      "namedPorts": [
+        {
+          "name": "http",
+          "port": 80
+        }
+      ],
+      "network": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/zones/us-central1-a/instanceGroups/ig-test-vmd",
+      "size": 0,
+      "subnetwork": "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1/subnetworks/default",
+      "zone": "https://www.googleapis.com/compute/v1/projects/test-area/zones/us-central1-a",
+      "id": "2986056884972096897",
+      "name": "ig-test-vmd",
+      "shortZone": "us-central1-a",
+      "shortRegion": "us-central1"
+    }
+  ],
+    "metadata": {
+    "recordCount": 1
+  }
+}
+```
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instancegroups</code>
+
+Retrieve a list of all instance groups in a given [environment](#administration-environments)
+
+
+Attributes | &nbsp;
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format
+`description`<br/>*string* | An optional description of this resource
+`fingerprint`<br/>*string* | A short sequence of bytes used to identify the named ports
+`kind`<br/>*string* | Type of the resource
+`namedPorts`<br/>*object* | Assigns a name to a port number. This allows the system to reference ports by the assigned name instead of a port number. Named ports apply to all instances in this instance group.
+`network`<br/>*string* | The network to which the instance group is connected to
+`selfLink`<br/>*string* | Server-defined URL for this resource
+`size`<br/>*integer* | The number of instances in the group
+`subnetwork`<br/>*string* | The sub-network within the network the instance group is connected to
+`zone`<br/>*string* | URL of the zone where the instance group resides
+`id`<br/>*UUID* | The id of the instance group
+`name`<br/>*string* | The display name of the instance group
+`shortZone`<br/>*string* | A short version of the zone name
+`shortRegion`<br/>*string* | A short version of the region name
```

Comment:
Minor, but we might want to order this alphabetically

Reply:
👍

Reply:
I tend to follow the way we get it from the API.

Reply:
It's JSON, so there is no guaranteed order

Reply:
will do alphabetical, and create a JIRA to review all existing docs to apply this new rule.

Reply:
Done